.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+name: Static analysis
+run-name: GitHub Actions for clang-tidy
+
+on: pull_request
+
+env:
+  UBUNTU_CODE_NAME: jammy
+  COMPILER_VERSION: 17
+  BOOST_VERSION: 1_77_0
+
+jobs:
+  clang-tidy:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+          submodules: 'recursive'
+
+      - name: Fetch base branch
+        run: |
+          git remote add upstream "https://github.com/${{ github.event.pull_request.base.repo.full_name }}"
```

Comment:
Right. It is unnecessary to point 2 remotes to the same repo since origin will already be pointing to the appropriate repo. Will remove it.

Reply:
Sorry, @VarunNagaraju I might have mislead you. I wanted to say that in case of a cross-fork PR we will need 2 remotes:
```
origin https://github.com/percona/percona-server.git 
fork origin https://github.com/<percona_user>/percona-server.git 
```
But in case of a PR within the same repo (say, when we merge `release-8.0.xx-yy` branch into `8.0` trunk) we will basically have
```
origin https://github.com/percona/percona-server.git 
fork https://github.com/percona/percona-server.git 
```
And my question was more about if git itself allows to have 2 remotes with different names pointing to the same repo.